### PR TITLE
Use original name of a library

### PR DESCRIPTION
**Pull request by ivankravets**

### CODE REVIEW

**Comment by fingolfin** [comment]
Isn't 2.6.3 a smaller version than 2.52? And why 2.6.3 and not 2.6.0 or any other 2.6.x ?

**Comment by ivankravets** [comment]
Please sorry, someone submitted manifest with a new version to @PlatformIO. I've just taken a loot at the latest releases, and see that 2.6.x were issued for Arduino IDE.

I removed from this PR changes related to "version" field and updated only the library name and description.

Thanks!

File: library.json [hunk 1]
```diff
@@ -1,11 +1,7 @@
 {
-  "name": "RCSwitch",
-  "description": "Use your Arduino or Raspberry Pi to operate remote radio controlled devices",
+  "name": "rc-switch",
+  "description": "Arduino lib to operate 433/315Mhz devices like power outlet sockets",
```
Comment (fingolfin) [inline library.json]: Why "lib" and not "library"

Comment (ivankravets) [inline library.json]: Need to ask owner of this repository. I copied it from https://github.com/sui77/rc-switch

**Pull request closed without merging**